src/json_deser.c

```diff
@@ -401,3 +414,140 @@ corto_int16 json_deserialize(corto_object o, corto_string s) {
     }
     return -1;
 }
+
+/*
+ * This method changes the id!
+ */
+static void json_splitParentAndName(char* id, char** parentName, char** name)
```

Comment:
Consider changing this name to `json_splitId`, as that is a more concise description of what you're parsing.